mtp_api/apps/prison/tests/test_views.py

```diff
@@ -488,6 +491,99 @@ def test_valid_prisoner_not_found_with_incorrect_prison_filter(self):
         self.assertEmptyResponse(response)
 
 
+class PrisonerAccountBalanceTestCase(AuthTestCaseMixin, APITestCase):
```

Comment:
Minor point, but given that the test setup for private and public prison test data seems mostly exclusive, and that no test method uses both, have you considered breaking this out into two test case classes?